README.md

```diff
@@ -201,17 +201,78 @@ To create a new Breakup Simulation just follow these four steps:
 3. Create your Breakup Simulation
 4. Run and get the Result!
 
-Example:
+### Example CMake integration:
+```cmake
+cmake_minimum_required(VERSION 3.16)
+project(FOO)
 
+# ------------------------- get the breakup model -------------------------
+# Enable ExternalProject CMake module
+include(FetchContent)
+
+# OPTIONAL: disable stuff not needed when used as a lib
+set(BUILD_BREAKUP_MODEL_TESTS OFF CACHE INTERNAL "")
+set(BUILD_BREAKUP_MODEL_SIM OFF CACHE INTERNAL "")
+
+FetchContent_Declare(
+        BreakupModelfetch
+        GIT_REPOSITORY git@github.com:esa/NASA-breakup-model-cpp.git
+        # GIT_TAG master
+)
+
+# Get Breakup Model source and binary directories from CMake project
+FetchContent_MakeAvailable(BreakupModelfetch)
+
+# OPTIONAL: Disable potential warnings
+# from the library target
+target_compile_options(breakupModel_lib PRIVATE -w)
+# from included headers
+get_target_property(propval breakupModel_lib INTERFACE_INCLUDE_DIRECTORIES)
+target_include_directories(breakupModel_lib SYSTEM PUBLIC "${propval}")
+
+# -------------------- define your target and link it ---------------------
+
+add_executable(fooExe foo.cpp)
+target_compile_features(fooExe PUBLIC cxx_std_17)
+target_link_libraries(fooExe PUBLIC breakupModel_lib)
+```
+
+### Example C++ Code integration:
 ```cpp
-    //RuntimeInput via the RuntimeInputSource Object (minmial Config: minL_c = 0.05 + inputSatellites)
+#include <breakupModel/input/RuntimeInputSource.h>
+#include <breakupModel/model/Satellite.h>
+#include <breakupModel/model/SatelliteBuilder.h>
+#include <breakupModel/simulation/BreakupBuilder.h>
+
+int main() {
+    // generate satellites to collide
+    SatelliteBuilder satelliteBuilder;
+    std::vector<Satellite> satellites{
+        satelliteBuilder
+            .setID(0)
+            .setPosition({0.,0.,0.})  // [m]
+            .setVelocity({0.,1.,0.})  // [m/s^2]
+            .setMass(500.)            // [kg]
+            .getResult(),
+        satelliteBuilder
+            .setID(1)
+            .setPosition({0.,0.,0.})  // [m]
+            .setVelocity({0.,0.,1.})  // [m/s^2]
+            .setMass(500.)            // [kg]
+            .getResult(),
+    };
```

Comment:
What happens if we put three satellites in this list?

Reply:
You get an exception telling you that the simulation does not know what you want from it ;)
The idea here was the following:
- If you pass exactly one satellite it assumes you want to simulate an explosion
- If you pass exactly two satellites is assumes you want to simulate a collision
- If you pass more (e.g. because you are dealing with a huge vector of satellites and do not want to copy around data) you need to also pass a filter that tells the simulation which elements of your vector are the relevant ones. You can then also specify collision or explosion via `BreakupBuilder::setSimulationType()` or the config.

Reply:
We should probably mention this prominently somewhere?

Reply:
The Readme states:

```yaml
simulationType: COLLISION         #Option (Alias): COLLISION (CO) or EXPLOSION (EX)
                                      #If not given type is determined
                                      # by number of input satellites 
```
We could extend this and/or add more text somewhere else. What do you suggest?

Reply:
```yaml
simulationType: COLLISION         #Option (Alias): COLLISION (CO) or EXPLOSION (EX)
                                  #If not given type is determined
                                  # by number of input satellites ( 1 - EX, 2 - CO).
```

But more importantly somewhere where the satelliteBuilder is used? so right here as a comment, I suppose 🤷

Reply:
And pretty much the same thing you wrote above :D

Reply:
like that? 734f84b

Reply:
yes plus suggestion I added above